core/src/test/java/io/grpc/internal/ExponentialBackoffPolicyTest.java

```diff
@@ -56,9 +56,8 @@ public double nextDouble() {
   @Test
   public void maxDelayReached() {
     long maxBackoffMillis = 120 * 1000;
-    policy.setMaxBackoffMillis(maxBackoffMillis)
-        .setJitter(0)
-        .setRandom(notRandom);
+    ExponentialBackoffPolicy policy = new ExponentialBackoffPolicy(notRandom,
+            TimeUnit.SECONDS.toMillis(1), maxBackoffMillis, 1.6, 0);
```

Comment:
This is a good reason to introduce a builder pattern instead of having a long ctor argument list.